general_rules/Robots-OPL.tex

```diff
@@ -47,4 +50,11 @@ \subsubsection{Audio output plug}\label{rule:roobt_audio_out}
 		\item It is the responsibility of the teams to plug in the transmitter before a test, to check the sound system, and to hand over the transmitter to next team.
 		\item Do not rely on the sound system! For fail-safe operation and interacting with operators make sure that the sound system is not needed, e.g., by having additional speakers directly on the robot.
 \end{compactitem}
-\end{enumerate}
\ No newline at end of file
+\end{enumerate}
+
+
+
+
+\subsubsection{Appearance}
+\label{rule:robots_appearance}
```

Comment:
I know marking any else than the robot has indeed been disallowed but markings on the robot itself have been fine, AFAIK.